src/Data/String/CodePoints.purs

```diff
@@ -50,6 +50,9 @@ newtype CodePoint = CodePoint Int
 derive instance eqCodePoint :: Eq CodePoint
 derive instance ordCodePoint :: Ord CodePoint
 
+instance showCodePoint :: Show CodePoint where
+  show cp = "(CodePoint " <> show (singleton cp) <> ")"
```

Comment:
This instance should definitely exist, but I think it would be better to print the contained `Int` in hex format. Do you know if that's possible now, or will we need to add a dependency?

Reply:
https://pursuit.purescript.org/packages/purescript-integers/3.2.0/docs/Data.Int#v:toStringAs would do the trick. I think, there is an advantage of displaying the Unicode representation. Maybe a combined version?
`(CodePoint 0x... - "∆")`. That way you can see if you have chosen the correct number for the symbol you wanted.

Reply:
I think it would be best to just use the `Int`, to keep it simple, and to avoid suggesting that there's anything more to a `CodePoint` than just an `Int`. People can use `singleton` if they want to see the code point as a string.